tests/queries/0_stateless/01069_database_memory.reference

```diff
@@ -5,4 +5,4 @@ CREATE DATABASE memory_01069\nENGINE = Memory()
 4
 3
 4
-CREATE TABLE memory_01069.file\n(\n    `n` UInt8\n)\nENGINE = File(\'CSV\')
+CREATE TABLE memory_01069.file\n(`n` UInt8\n)\nENGINE = File(\'CSV\')
```

Comment:
I suppose we need to remove `\n` before `(` and after `)`. Or restore previous behaviour.

Reply:
Now you can see that the test results only differ in trailing whitespaces, as expected.